examples/prisma-orm/README.md

```diff
@@ -0,0 +1,156 @@
+# Prisma ORM + Next.js starter
+
+This repository provides a boilerplate to quickly set up a Next.js application with [Prisma Postgres](https://www.prisma.io/postgres) and [Prisma ORM](https://www.prisma.io/orm) for database operations. It includes an easy setup process and example routes to add, view, and list posts, demonstrating basic CRUD operations with Prisma ORM.
+
+## How to use
+
+Run [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app) using your preferred package manager:
+
+```bash
+# Using npm
+npx create-next-app --example prisma-orm prisma-orm-app
+```
+
+<details>
+
+<summary>Expand for <code>yarn</code>, <code>pnpm</code> or <code>bun</code></summary>
+
+```bash
+# Using yarn
+yarn create next-app --example prisma-orm prisma-orm-app
+
+# Using pnpm
+pnpm create-next-app --example prisma-orm prisma-orm-app
+
+# Using bun
+bunx create-next-app --example prisma-orm prisma-orm-app
+```
+
+</details>
+
+## Getting started
+
+Follow these steps to quickly set up the project and start using Prisma ORM with Next.js.
+
+### 1. Create a Prisma Postgres instance
+
+Create a Prisma Postgres database instance using [Prisma Data Platform](https://console.prisma.io):
+
+1. Navigate to [Prisma Data Platform](https://console.prisma.io).
+2. Click **New project** to create a new project.
+3. Enter a name for your project in the **Name** field.
+4. Inside the **Prisma Postgres®** section, click **Get started**.
+5. Choose a region close to your location from the **Region** dropdown.
+6. Click **Create project** to set up your database. This redirects you to the database setup page.
+7. In the **Set up database access** section, copy the `DATABASE_URL`. You will use this in the next steps.
+
+#### Setup your `.env` file
+
+1. Create an `.env` file:
+   ```bash
+   touch .env
+   ```
+2. Update the `.env` file by replacing the existing `DATABASE_URL` value with the one you previously copied. It will look similar to this:
```

Comment:
````suggestion
You now need to configure your database connection via an environment variable.

First, create an `.env` file:

```bash
touch .env
```

Then update the `.env` file by replacing the existing `DATABASE_URL` value with the one you previously copied. It will look similar to this:
````